practice-app/website/api/art_galleries.py

```diff
@@ -0,0 +1,22 @@
+import requests
+from flask import Blueprint, request, jsonify
+
+
+art_galleries = Blueprint('art_galleries', __name__)
+
+
+@art_galleries.route('art_galleries')
+def get_art_galleries():
+    galleries = []
+
+    for i in range(5):
+        data = requests.get(
+            'https://random-data-api.com/api/address/random_address'
+        ).json()
```

Comment:
Maybe the status code might be checked before proceeding to make sure that the provider is available.

A line like `if 200 <= status_code < 300` might be helpful.